sdk/python/kfp/compiler/test_data/components/unsupported/output_named_tuple.py

```diff
@@ -22,7 +22,7 @@
 
 @component
 def output_named_tuple(
-    artifact: Input[Dataset]
+    dummy: str,
```

Comment:
nit: remove unused imports `Dataset` and `Input`

Reply:
Done.